Example/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>2.7.0</string>
 	<key>CFBundleVersion</key>
-	<string>116</string>
+	<string>2</string>
```

Comment:
Should be `117` I guess.

Reply:
That's correct. It picked up the version from Example and set the incremented one to the whole project. Fixed it in https://github.com/mapbox/mapbox-navigation-ios/pull/4037, because release branches are protected from rebases.